channeldb/channel.go

```diff
@@ -2111,6 +2111,11 @@ func (c *OpenChannel) AppendRemoteCommitChain(diff *CommitDiff) error {
 			return ErrChanBorked
 		}
 
+		// Delete the contents of unsignedAckedUpdatesKey.
+		if err := chanBucket.Delete(unsignedAckedUpdatesKey); err != nil {
```

Comment:
Makes a lot of sense deleting it here btw 👍  Not sure why it wasn't done before

Reply:
yes should have done it before, but I thought we could get away with deleting it upon revoking again